src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/OverrideEqualsAndOperatorEqualsOnValueTypes.cs

```diff
@@ -66,10 +66,12 @@ public override void Initialize(AnalysisContext analysisContext)
                     //  2. Do not fire for enumerators.
                     //  3. Do not fire for value types without members.
                     //  4. Externally visible types by default.
+                    //  5. Do not fire for ref struct.
                     // Note all the descriptors/rules for this analyzer have the same ID and category and hence
                     // will always have identical configured visibility.
                     if (!namedType.IsValueType ||
                         namedType.TypeKind == TypeKind.Enum ||
+                        (namedType.TypeKind == TypeKind.Struct && namedType.IsRefLikeType) ||
```

Comment:
I don't think there is any other type can be `RefLikeType` but I went for the belt and braces option here.